src/views/product-integrations-landing/components/searchable-integrations-list/index.tsx

```diff
@@ -318,6 +319,19 @@ export default function SearchableIntegrationsList({
 							)
 						)
 					})}
+					{/* Render x-tags for Types */}
+					{types.map((e, i) => {
+						const checked = typesCheckedArray[i]
+						return (
+							checked && (
+								<Tag
+									key={e.id}
+									text={e.plural_name}
+									onRemove={makeToggleTypeHandler(i, e.name)}
+								/>
+							)
+						)
+					})}
```

Comment:
Pack tags were not displaying in this list.

![CleanShot 2023-05-17 at 16 26 44@2x](https://github.com/hashicorp/dev-portal/assets/2105067/97de465e-22eb-48bf-b93c-a53f20dd8c1d)